src/assets/scss/views/statistics/_statistics.scss

```diff
@@ -0,0 +1,31 @@
+.md-drppicker {
```

Comment:
This should be put in a dedicated files for the date/time picker and imported globally